party-type-individual-processing-eninge/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "airbnb-typescript/base"
+  ],
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "rules": {
+    "linebreak-style": [
+      "error",
+      "windows"
+    ] // changes the file to CRLF
```

Comment:
works locally for your development environment, but the container will run on linux so this will trigger an error in the CI eventually

Reply:
eslint is dev dependency - so shouldn't have a problem for run-time. Should possibly gitignore this file, as it won't work on your dev env either.